### ISSUE TITLE
"arrow-parens" w/ async arrow function w/ arguments

### ISSUE THREAD

**Issue by jonathanong** [babel/eslint-plugin-babel#25]
```js
const fn = async (ctx, next) => {

}
```

Error:

```
error  Expected parentheses around arrow function argument  arrow-parens
```

is this an error on your side or is this invalid syntax? using `eslint@1.8.0` and `babel-eslint@4.1.4`

**Comment by hzoo** [comment]
It should be valid - did you use `babel/arrow-parens`? Looks like we already have a rule here that should support it?

**Comment by jonathanong** [comment]
no i used eslint's default. are you saying i have to use `babel-eslint`'s rule?

**Comment by hzoo** [comment]
Yeah this is similar to the other ones - since eslint doesn't know about newer syntax the default rules won't be able to support it.

**Comment by jonathanong** [comment]
ah got it. i think i've gotten this repo confused with babel-parser as well. thanks
